lib/tokenable/revokeable.rb

```diff
@@ -0,0 +1,33 @@
+module Tokenable
+  module Revokable
+    extend ActiveSupport::Concern
+
+    def token_revoked?(revoke_token)
```

Comment:
You're not actually checking if the token is reokved, the exact opposite. Rather, we want to check if the `secret` is valid.